fury/tests/test_colormap.py

```diff
@@ -115,3 +115,10 @@ def test_lab2rgb():
     res = np.round(colormap._lab2rgb(lab_color))
     res[res < 0] = 0
     npt.assert_array_almost_equal(res, expected)
+
+
+def test_hex_to_rgb():
+    expected = np.array([0.99609375, 0.99609375, 0.99609375])
+    hexcode = "#FFFFFF"
+    res = colormap.hex_to_rgb(hexcode)
+    npt.assert_array_almost_equal(res, expected)
```

Comment:
Do not forget to add a test when `hexcode = ffffff` (without #)

Reply:
Done